### ISSUE TITLE
LocalizedName property in addition to Name property? [Feature Request]

### ISSUE THREAD

**Issue by ollypsilon** [microsoft/accessibility-insights-windows#1574]
I wonder if it's within scope of AI tool, if not please close this request.

In multi-language installations MS Access gives localized names for same object depending on application language.
E.g. with Access started with English UI it's "Table Linked to Access" and with German UI it is "Mit Access verknüpfte Tabelle".
So, I wonder if there could be a LocalizedName property like it is for LocalizedControlType and Name property contains not localized name (if this exists in parallel to localized name).
![image](https://user-images.githubusercontent.com/52780811/225052046-122c38fe-5b59-40c4-b258-a934d62d725e.png)
![image](https://user-images.githubusercontent.com/52780811/225054503-e35b3f85-2a45-4557-a18c-43deb27216a7.png)

**Comment by codeofdusk** [comment]
Out-of-scope: the Name property is assumed to be localized. If there is a serious need for an out-of-band Name, this would need to be added as a UIA custom property which can be read by Accessibility Insights.
